test/e2e/kubectl/logs.go

```diff
@@ -356,5 +358,50 @@ var _ = SIGDescribe("Kubectl logs", func() {
 
 		})
 	})
+})
+
+var _ = SIGDescribe("kubectl logs with specific stream", feature.PodLogsQuerySplitStreams, framework.WithFeatureGate(features.PodLogsQuerySplitStreams), func() {
+	f := framework.NewDefaultFramework("kubectl-logs-stream")
+	f.NamespacePodSecurityLevel = admissionapi.LevelBaseline
+	defer ginkgo.GinkgoRecover()
+
+	var c clientset.Interface
+	var ns string
+	ginkgo.BeforeEach(func() {
+		c = f.ClientSet
+		ns = f.Namespace.Name
+	})
 
+	ginkgo.Describe("specific log stream", func() {
+		podName := "log-stream"
+		ginkgo.BeforeEach(func() {
+			ginkgo.By("creating a pod")
+			e2ekubectl.RunKubectlOrDie(ns, "run", podName, "--image="+imageutils.GetE2EImage(imageutils.BusyBox), "--restart=Never", podRunningTimeoutArg, "--", "/bin/sh", "-c", "echo out1; echo err1 >&2; tail -f /dev/null")
```

Comment:
Would it make sense expecting out1 and err1 if the stream option is all, when this feature gate is enabled?.

Reply:
Do you know where these tests would run in the CI?

the jobs I triggered are unrelated it seems.